### PR TITLE
Feat : Filters added date, trending

### PR DESCRIPTION
**Pull request by t007rushi**
# Filters 
- Trending Posts
# sort by  latest and old ones

### CODE REVIEW

File: src/utility/categoryFilterUtil.js [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+      case "✨Latest":
+        return followingsPosts.sort(
+          (first, second) =>
+            first.createdAt - second.createdAt
+        );
+      case "🐼Old":
+        return followingsPosts.sort(
+          (first, second) =>
+            second.createdAt - first.createdAt
+        );
```
Comment (JantuDeb) [inline src/utility/categoryFilterUtil.js:16]: You can create constant for these `case`

**Review by narharikale — APPROVED** [review]
good code structure 
liked firebase implementation
good to go

**Review by JantuDeb — APPROVED** [review]
Overall the code looks good

File: src/pages/Auth/Signup/Signup.jsx [hunk 2]
```diff
@@ -86,6 +89,7 @@ export const Signup = () => {
           Create Password
           <input
             type="password"
+            autocomplete="on"
```
Comment (MarkVed17) [inline src/pages/Auth/Signup/Signup.jsx:92]: Useful attribute!

**Review by MarkVed17 — APPROVED** [review]
Everything seems good to merge!
Good work @t007rushi 🚀

File: src/pages/Auth/Signup/Signup.jsx (hunk 2 above)
Comment (horsemaker) [reply src/pages/Auth/Signup/Signup.jsx:92]: But, autocomplete for password? 🤔

File: src/pages/Home/Home.jsx [hunk 3]
```diff
@@ -1,85 +1,59 @@
...
+
+  const selectCategoryHandler = (value) => {
+    setCurrentCategory(value);
+  };
 
   useEffect(() => {
     dispatch(getAllPosts());
     dispatch(getAllUsers());
     dispatch(getAllComments());
   }, []);
```
Comment (horsemaker) [inline src/pages/Home/Home.jsx:34]: Have you taken care of cleanups too?

**Review by horsemaker — APPROVED** [review]
The code looks modular and clean. 👌
Good work Rushikesh.

File: src/pages/Auth/Signup/Signup.jsx (hunk 2 above)
Comment (t007rushi) [reply src/pages/Auth/Signup/Signup.jsx:92]: yes its giving me warning about that thing so added

File: src/pages/Home/Home.jsx (hunk 3 above)
Comment (t007rushi) [reply src/pages/Home/Home.jsx:34]: yup will do that next

**Pull request merged**